### PR TITLE
[DECLUTTER FAQ] rework Installation

### PR DESCRIPTION
**Pull request by MaxHillebrand**
In this ready for review branch, I have read through the Installation FAQ, fixed up some minor grammar and typos, edited wrong statements, and removed one redundant question.

### CODE REVIEW

File: docs/FAQ/FAQ-Installation.md [hunk 1]
```diff
@@ -126,19 +133,21 @@ It will also be announced on [Twitter](https://twitter.com/wasabiwallet), [Reddi
 
 You can download the software build for the different operating systems on the main [website](https://wasabiwallet.io) or better over [Tor](http://wasabiukrxmkdgve5kynjztuovbg43uxcbcxn6y2okcrsg7gb6jdmbad.onion).
 Make sure you also download the signatures of the build and verify them with [zkSNACKs' PGP public key](https://github.com/zkSNACKs/WalletWasabi/blob/master/PGP.txt).
+To update Wasabi, install the new version the same way as usual, it will replace the old version.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-Installation.md]: This doesn't add any information.
```suggestion
```

File: docs/FAQ/FAQ-Installation.md [hunk 2]
```diff
@@ -126,19 +133,21 @@ It will also be announced on [Twitter](https://twitter.com/wasabiwallet), [Reddi
...
 All Wasabi network traffic goes via Tor by default - no need to set up Tor yourself.
 If you do already have Tor, and it is running, then Wasabi will try to use that first.
 
 You can turn off Tor in the Settings.
-Note that in this case you are still private, except when you CoinJoin and when you broadcast a transaction.
-In the first case, the coordinator would know the links between your inputs and outputs based on your IP address.
-In the second case, if you happen to broadcast a transaction of yours to a full node that is spying on you, it will know the link between your transaction and your IP address.
+Be careful, as this will compromise your privacy.
+When you CoinJoin over clearnet, the coordinator would know the links between your inputs and outputs based on your IP address.
+When you broadcast a transaction ofer clearnet, the connected full node will know the link between your transaction and your IP address.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-Installation.md]:
```suggestion
When you broadcast a transaction over clearnet, the connected full node will know the link between your transaction and your IP address.
```

File: docs/FAQ/FAQ-Installation.md [hunk 3]
```diff
@@ -186,27 +195,39 @@ Wasabi has [reproducible and deterministic builds](/using-wasabi/DeterministicBu
...
+```
+:::
+
+:::details
+### How can I run Wasabi headless daemon without GUI?
+
+To use Wasabi's command line tools on Windows you have to use `wassabeed.exe` that is inside your `Program Files\WasabiWallet`.
+On Linux and macOS you can use the same software that you use for launching the GUI (`wassabee`).
+One can run it with the same command: `wassabee mix --wallet WalletName`.
+Optionally you can add a `--keepalive` flag that will make sure your daemon will run forever and mixes forever.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-Installation.md]: Why add back this? it was removed three days ago https://github.com/zkSNACKs/WasabiDoc/pull/643

```suggestion
```

Comment (MaxHillebrand) [reply docs/FAQ/FAQ-Installation.md]: Oh now I understand what was happening... it seems my local repo was out of date, and when I pushed this branch, it showed that there were conflicts with master - which was removing of this question. I fixed the conflict wrongly, by keeping the question in, and not removing it. 

Sorry about it, good catch @yahiheb

File: docs/FAQ/FAQ-Installation.md [hunk 4]
```diff
@@ -186,27 +195,39 @@ Wasabi has [reproducible and deterministic builds](/using-wasabi/DeterministicBu
...
 :::
```
Comment (MaxHillebrand) [inline docs/FAQ/FAQ-Installation.md]:
```suggestion
```

**Pull request merged**